commands/update-check.go

```diff
@@ -1,34 +1,35 @@
 package commands
 
 import (
-	"time"
-	"strings"
 	"context"
 	"fmt"
+	"strings"
+	"time"
 
+	"github.com/activecm/rita/config"
+	"github.com/activecm/rita/resources"
 	"github.com/blang/semver"
-	"github.com/urfave/cli"
 	"github.com/google/go-github/github"
-	"github.com/activecm/rita/resources"
-	"github.com/activecm/rita/config"
 	log "github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
 )
 
 //Strings used for informing the user of a new version.
-var informFmtStr string = "\nTheres a new %s version of RITA %s available at:\nhttps://github.com/activecm/rita/releases\n"
-var versions = []string{ "Major", "Minor", "Patch" }
+var informFmtStr = "\nTheres a new %s version of RITA %s available at:\nhttps://github.com/activecm/rita/releases\n"
```

Comment:
Golint wanted "string" removed here because it is unnecessary and can be inferred from the right-hand side of the assignment operator.